templates/zerver/help/delete-a-message.md

```diff
@@ -0,0 +1,7 @@
+# Delete a message
+
+{!admin-only.md!}
```

Comment:
The text in the admin-only macro ("This feature can only be controlled by organization administrators") is a little strange for this particular feature, since it's not really a setting that can be controlled but more of an action that can be done.  Maybe it's worth thinking about rewriting the text in that macro, or maybe we just write a different macro like `{!admin-only-action.md!}` for actions that only admins can do.